qendpoint-frontend/src/components/SparqlEndpoint/plugins/queryPlan.ts

```diff
@@ -48,8 +48,10 @@ export default class QueryPlanPlugin {
       const plan = json.plan
       const plnEl = buildPlan(plan)
       console.log(plnEl)
-      const container = document.createElement('ul')
-      container.appendChild(plnEl)
+      const container =  document.createElement("pre")
+
+     container.innerText = JSON.stringify(plan, null, 2)
+

```

Comment:
Can you apply the formatter? `npm run format`

Reply:
Formatter didn't seem to do anything, but IntelliJ highlighted the issues and I fixed them by hand.